src/test/java/core/basesyntax/service/impl/CsvFileReaderImplTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileReader;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class CsvFileReaderImplTest {
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
```

Comment:
```suggestion
    public ExpectedException expectedException = ExpectedException.none();
```
the same for other similar cases